File: src/javas/algorithms/Bubble_Sort.java
package javas.algorithms;

public class Bubble_Sort {
    public static void main(String[] args) {

        int temp;
        int [] srt = {19,30,4,40,1};
        //number of round = array.length -1

        for (int i=0;i<srt.length-1;i++){  //number of round
            for (int j=0;j<srt.length-1;j++){ //Two adjustment element

                if (srt[j]>srt[j+1]){
                    temp=srt[j];
                    srt[j]=srt[j+1];
                    srt[j+1]=temp;

                }
            }
        }
        System.out.print("Bubble Sort ArrayList: ");
        for (int l=0;l<srt.length;l++) {
            System.out.print(srt[l]+" ");
        }


    }
}

File: src/Kotlin/algorithm/Bubble_Sort.kt
package Kotlin.algorithm

fun main() { 
    val srt = arrayOf(19,30,4,40,1) //1,4,19,30,40
    var temp=0


    for (i in 0 ..srt.size){  //number of round
        for (j in 0 until srt.size-1){//Two adjustment element

            if (srt[j]>srt[j+1]){
                temp =srt[j]
                srt[j]=srt[j+1]
                srt[j+1]=temp

            }
        }

    }
    for (i in 0 until  srt.size){
        print("${srt[i]} " )
    }

}